src/fxcorrelator_bengops.py

```diff
@@ -1355,13 +1363,13 @@ def spead_issue_all(corr, beams=[], from_fpga=True):
     :param from_fpga:
     :return:
     """
-    spead_static_meta_issue(corr, beams=beams)
+    spead_static_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
     spead_passband_meta_issue(corr, beams=beams)
```

Comment:
It seems that we might actually need to be able to change the centre frequency and bandwidth so this functionality should not be scrapped at the moment.